### ISSUE TITLE
The thing with the tab replacement by four spaces

### ISSUE THREAD

**Issue by Nerdmind** [erusev/parsedown#508]
Hello. I would also (#363) find it good if tabulators remain tabluators. Especially for the indentation of code one has actually only to use tabulators (everything else is semantically wrong and inflexible). What would happen if I defined in CSS that a tab should be displayed in the HTML pre-tags like 12 spaces (`tab-size: 12;`)? This does not work because the CSS does not find any tabulators, but only four hard blanks. If the user places a tab anywhere within its text or source code, the tab should always remain a tab. Especially within HTML pre-tags where I may post my source code which is semantically correct indent with tabs! If the markdown parser replaces my tabs, the code isn't anymore THE code I've inserted.

**If this is my original code which I want to put into my markdown (tab indentation) …**
```
class Foobar {
	protected $propertyA = 1;
	protected $propertyB = 2;
}
```

**… Parsedown will make this, which is not my code anymore:**
```
class Foobar {
    protected $propertyA = 1;
    protected $propertyB = 2;
}
```

If you compare the two code examples, you'll see that GitHub also retains the tab indentation in the first example and uses a width like 8 spaces for displaying them (8 is the default width in HTML pre-tags for tabs). And this is EXACTLY the same code I entered in the issue editor. The second example is not my code because tabs were replaced by spaces. This is inflexible if someone would copy the code into his IDE in which he uses, for example, a width like 15 spaces for displaying tabs.

**Shortly:** Tabulators should remain tabulators. :)

**Comment by Nerdmind** [comment]
In the meantime, I've hacked the library to keep tabulators present in the output and it seems to work fine. I'd to change code inside 3 methods: `lines`, `blockCode` and `blockCodeContinue`. In the `line` method, I've completely removed the `if` condition which checks for tabulators and deletes them.

Within the last two methods, I've modified the `if` conditions to check both against four spaces and a tabulator. The patch is for the latest release version 1.6.2 (by the way: you have forgotten to change the version number in the `version` constant).

**Patch:** [parsedown-keep-tabs-1.6.2.patch.zip](https://github.com/erusev/parsedown/files/1041653/parsedown-keep-tabs-1.6.2.patch.zip)

**Comment by petk** [comment]
Hello, this would be a very useful addition. Usually I don't like tabs anywhere but Makefile, for example, by default requires to have tabs... And having a Makefile code in Markdown makes the HTML parsed code nonfunctional. For tutorials etc :)

**Comment by aidantwoods** [comment]
I know it's a while later, but this should be fixed in the `2.0.x` branch, which goes to quite a bit of effort to track not only tabs, but correct indentation offsets through sub-parsing of blocks :)

<img width="850" alt="Screenshot 2019-04-07 at 16 12 54" src="https://user-images.githubusercontent.com/3288888/55685598-0ead3980-5950-11e9-9d6f-6734e9cf41c5.png">
